coffee/room.coffee

```diff
@@ -84,8 +84,7 @@ class palava.Room extends @EventEmitter
   #
   join: (status = {}) =>
     @joinCheckTimeout = setTimeout ( =>
-      @emit 'join_error', 'Not able to join room'
-      @leave() # TODO ?
```

Comment:
The rest makes sense, but is there any reasonable situation in which we do not want to send the leave message when this timeout happens? I have the feeling this should stay?

Reply:
It seems that the only situation the join_error is thrown is, when no "joined_room" event has been received (in time) --> which might be an indication that the client never actually joined.

Is the signaltower able to deal with orphaned "leave" events (= ignore)? Then we could also say, we just send it all the time

Reply:
yes it can but it will send you an error message back (`{event: "error", description: "You are not currently in a room, so you can not leave it"}`) which would be ok I guess?

Reply:
yep!